OasisDEX/oasis-borrow
# New sidebar design in adjust stop loss

# [New sidebar design in adjust stop loss](https://app.shortcut.com/oazo-apps/story/4505/implement-adding-stop-loss-form-according-to-new-design)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>
- added new sidebar design in adjust stop loss form
  
## How to test 🧪
  <Please explain how to test your changes>
- use feature flag `stopLossWriteEnabled` and `newComponentsEnabled` and go to protection tab
- verify all possible scenarios to ensure correct state handling between transition etc.
- double check whether setting up stop loss works correctly on the on design
